### ISSUE TITLE
Phase association issue (regression)

### ISSUE THREAD

**Issue by mmesim** [swiss-seismological-service/scdetect#125]
Well, I was testing [feature/iss-118](https://github.com/swiss-seismological-service/scdetect/tree/feature/iss-118) when I noticed something weird. When I run `scdetect-cc` on multiple stations & multiple channels and setting the `minimumArrivals` to 8 (for example) in the `template.json`, the code returns all possible detections and ignores completely this value. Same when I run `scdetect-cc` on master. 

Then I tried to set the value in the module (`scconfig`) and I got this

``` bash
15:36:07 [error/DETECT] Invalid configuration: 'minimumArrivals': 8. Must be < 0 or >= 1
Try --help for help
```

I pulled today the most recent master from seiscomp github page, but I have no idea if this issue is related to that.

**Comment by mmesim** [comment]
The issue is definitely related to `scdetect-cc` and the current master branch. 

I ran a few examples with an older version of `scdetect-cc` and it worked fine. However I still get  the same error when trying to fix `Minimumarrivals` in `scconfig`. 

When I switched to the latest master the `MinimumArivals` set in `templates.json` is ignored.

**Comment by mmesim** [comment]
[input.files.zip](https://github.com/swiss-seismological-service/scdetect/files/9350087/input.files.zip)

In the zip you can find all the input files and the expected results.

**Comment by damb** [comment]
Perfect, @mmesim. Thanks a lot.

**Comment by mmesim** [comment]
> Please make sure the spelling of the configuration parameters is correct. E.g. instead of `MinimumArrivals` it must be `minimumArrivals` in the `templates.json`. See also: https://scdetect.readthedocs.io/en/stable/base/Template-Configuration.html#detector-configuration-parameters.
> 
> In the `templates.json` unknown configuration parameters are simply ignored.

I confirm that I misspelled `minimumArrivals` here. The .json file is fine.

**Comment by damb** [comment]
Actually, there were two issues:

- First of all, there was an issue with the phase association introduced with a1d8ce843ac81a09b82cc4930fc60a7ca639a80f. This should be fixed with #126.
- Besides, the default module configuration parameter wasn't validated properly such that the `scdetect-cc` stopped with the error message from above. Due to simplicity I fixed this one with #126, as well.
